Rakefile

```diff
@@ -18,11 +18,11 @@ def version_select
 end
 
 def xcpretty(cmd)
-  if `which xcpretty` && $?.success?
-    sh "set -o pipefail && #{cmd} | xcpretty -c"
-  else
+  # if `which xcpretty` && $?.success?
```

Comment:
forgot to clean this out?

Reply:
@krzysztofzablocki I was trying to get to the root cause of the failing tests—looks like I was missing an `import Foundation` in `ProtocolComposition.swift` which was causing the SwiftTemplate tests to fail to compile. Should be fixed now!